Jetty 12.0.x document modules (#9151)
eclipse/jetty.project: documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-eeN-deploy.adoc

//
// ========================================================================
// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
//
// This program and the accompanying materials are made available under the
// terms of the Eclipse Public License v. 2.0 which is available at
// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
// which is available at https://www.apache.org/licenses/LICENSE-2.0.
//
// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
// ========================================================================
//

[[og-module-eeN-deploy]]
===== Module `{ee-all}-deploy`

include::{JETTY_HOME}/modules/{ee-current}-deploy.mod[tags=description]

Deployment is managed via a `DeploymentManager` component that watches a directory for changes.

> (See xref:og-deploy[how to deploy web applications] for more information.)

Reviewer: Drop parens.

Adding files or directories to this monitored directory will cause the `DeploymentManager` to deploy them as web applications; updating files already existing in this monitored directory will cause the `DeploymentManager` to re-deploy the corresponding web application; removing files in this monitored directory will cause the `DeploymentManager` to "undeploy" the corresponding web application.
(You can find a more detailed discussion of these rules in the xref:og-deploy-rules[deployment rules] section.)

> Multiple versions of this module exist (`{ee-all}-deploy`) to support each EE platform's version of the Java Servlet specification.

Reviewer: `EE` -> `Jakarta EE`

Let's standardize on "Jakarta EE" everywhere, rather than just "EE".

Author: Sounds good to me.

> Jetty's configuration properties are nearly identical across these versions; the module file for `{ee-current}-deploy` is as follows:

Reviewer: It's not the module file as a whole, just the ini-template section.
So perhaps: 

```suggestion
Jetty's configuration properties are nearly identical across these versions; the configuration properties for the `{ee-current}-deploy` Jetty module are:
```
